overlord/snapstate/service_control.go

```diff
@@ -0,0 +1,138 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
doesn't this belong in the  servicestate package instead?

Reply:
there's an argument to be made at a later point Start/StopServices themselves should move there.  Given that we have a single TaskRunner nowadays this is a bit less of hassle than in the past.

Reply:
I wasn't sure if we wanted to keep servicestate around in the future? It's a bit hollow and doesn't have a manager. Do you envision actually having a manager there (with service-control, and later start/stop services handlers)?

Reply:
given the complexity that will come from intersnap dependencies not having to have all of that in snapstate seems a win, so yes